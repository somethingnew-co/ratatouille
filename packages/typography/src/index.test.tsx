import React from 'react';
import { ThemeProvider } from 'styled-components';
import { theme } from '../test/setup';
import { Text } from './index';
import { render } from '@testing-library/react';
import 'jest-styled-components';

test('it works', () => {
  const { container } = render(<ThemeProvider theme={theme}>
    <>
      <Text>Hello World</Text>
      <Text typeStyle="h1" as="h1">Testing H1</Text>
      <Text typeStyle="h2" as="h2">Testing H2</Text>
    </>
  </ThemeProvider>);
  expect(container).toMatchSnapshot();
})
;
