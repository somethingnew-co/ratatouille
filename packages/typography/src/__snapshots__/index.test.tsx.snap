// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  font-size: 16px;
}

.c1 {
  color: red;
}

@media screen and (min-width:40em) {
  .c0 {
    font-size: 24px;
  }
}

<div>
  <p
    class=""
  >
    Hello World
  </p>
  <h1
    class="c0"
    font-size="16,24"
  >
    Testing H1
  </h1>
  <h2
    class="c1"
    color="red"
  >
    Testing H2
  </h2>
</div>
`;
